tests/modules/bakta/test.yml

```diff
@@ -0,0 +1,25 @@
+## TODO nf-core: Please run the following command to build this file:
+#                nf-core modules create-test-yml bakta
+- name: bakta
+  command: nextflow run ./tests/modules/bakta -entry test_bakta -c tests/config/nextflow.config -stub-run
+  tags:
+    - bakta
+  files:
+    - path: output/bakta/test.embl
+      md5sum: d41d8cd98f00b204e9800998ecf8427e
```

Comment:
all the test output files are emtpy. Can the tool not be run with the avaible test data?

Reply:
I realize this is not optimal, but Bakta requires a 40GB+ input database, so I had to go with `-stub-run`. 

If its any consolation, I've implemented a test on my end that uses this module. https://github.com/bactopia/bactopia/blob/dsl2/subworkflows/local/bakta/test.yml (this done using a local runner for GitHub Actions, so its actually testing with the 40GB database)

Reply:
I see. OK then I'll approve the PR